go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/MichaelS11/go-hx711
+
+go 1.13
+
+require (
+	github.com/stianeikeland/go-rpio/v4 v4.4.0
+	periph.io/x/periph v3.6.2+incompatible
+)
```

Comment:
Don't know if we really need the go.mod file

Reply:
Better to have that one for the people who use modules, doesn't hurt if used with `$GOPATH`

At least if periph goes up one version - there won't be any surprises.